chaincode/mockstub_test.go

```diff
@@ -433,7 +433,9 @@ func (stub *MockStub) GetArgsSlice() ([]byte, error) {
 }
 
 func (stub *MockStub) setTxTimestamp(time *timestamp.Timestamp) {
-	stub.TxTimestamp = time
+	// Using a sequential timestamp make the tests' output determinist.
```

Comment:
What about adding a comment here, to explain that the goal of the +1 is to make sure the seed varies between different calls to mockInvoke within a same test?

Reply:
Will do yes.